core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -594,6 +713,7 @@ private CarbonCommonConstants() {
 
   /**
    * This property will determine the loads of how many days can be compacted.
+   * TODO: dynamic
```

Comment:
?

Reply:
removed, this property should can be dynamic configurable